dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM nginx:latest
+
+RUN apt-get update && apt-get upgrade -y
```

Comment:
Due to this..cache is created so we can remove it to reduce the size

Reply:
> Due to this..cache is created so we can remove it to reduce the size

well going on alpine is great but using this `rm -rf /var/cache/apk/*` is not recommended, as doing this will make it impossible to use `apk` effectively ahead. And you are actually preventing anyone from using your image as a base image and making modifications with `apk`. At least this is what happens with `apt` for sure and apk also as far as I know. 
If the maintainers can ensure that no one will use the image as base images in future we can go for it. But it only save 200 bytes and saving such small memory over accessibility of images is a bad trade off.
@romain-fontugne what are your views on it.

Reply:
Also, had it been `apt` we would have used `apt-get clean` but apk does it by default.

>clean: clears out the local repository of retrieved package files. It removes everything but the lock file from /var/cache/apt/archives/ and /var/cache/apt/archives/partial/.

Reply:
yeah! I used `alpine` which help.
Yep that ok size is so small 
Refer this - https://github.com/sgerrand/docker-alpine-r/blob/master/Dockerfile
https://stackoverflow.com/questions/49118579/alpine-dockerfile-advantages-of-no-cache-vs-rm-var-cache-apk

Reply:
Well both can be a example where they might not use image  as base images, is all I can say about this. And definitely we are moving to alpine. Thanks for collaborating.